packages/base/src/api/auth.ts

```diff
@@ -22,14 +23,21 @@ export async function authenticate(authenticationData: UserPass, poolData: PoolD
   const userData = { Username: authenticationData.Username, Pool: userPool };
   const cognitoUser = new CognitoUser(userData);
 
+  try {
+    return await retry(() => doAuthenticate(cognitoUser, authenticationDetails), { retries: 3 });
```

Comment:
I guess we do not want to filter on wich error we retry or not to prevent the weird one we had?

Reply:
Thing is we are seeing a "Incorrect username or password" when we believe it was related to throttling, so I'd rather be cautious and retry on all.